example/index.html

```diff
@@ -134,6 +134,34 @@ <h4> Posts: </h4>
             <button id="postThread" type="button" class="btn btn btn-primary">Post</button>
           </div>
 
+          <!-- GhostChat Start -->
```

Comment:
Did you have a chance to look at if this can be removed and just add an additional radiobutton to the threads code?

Reply:
Yeah that'll be finished today